x-pack/plugins/cases/public/components/user_actions/index.test.tsx

```diff
@@ -456,63 +418,6 @@ describe(`UserActions`, () => {
     expect(wrapper.find(`[data-test-subj="add-comment"] textarea`).text()).toBe(newComment);
   });
 
-  it('it should persist the draft of new comment while description is updated', async () => {
```

Comment:
moved this test to `x-pack/plugins/cases/public/components/case_view/case_view_page.test.tsx`